code/datums/action.dm

```diff
@@ -126,25 +136,28 @@
 
 /datum/action/item_action/Trigger(attack_self = TRUE) //Maybe we don't want to click the thing itself
 	if(!..())
-		return 0
+		return FALSE
 	if(target && attack_self)
 		var/obj/item/I = target
 		I.ui_action_click(owner, type)
-	return 1
+	return TRUE
 
 /datum/action/item_action/ApplyIcon(obj/screen/movable/action_button/current_button)
+	current_button.cut_overlays()
```

Comment:
Looking at line 156. It is not even needed to have this here. It'll be cut already there and on line 110 the parent cuts it if you were to go into the else.
That or line 156 can be gone and this statement can be put in the `if(use_itemicon)` scope